text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -373,8 +399,17 @@ def apply_layerwise_lr_decay(
     parameter_group_names = {}
     parameter_group_vars = {}
     decay_param_names = get_weight_decay_param_names(model)
-
+    norm_param_names = get_norm_layer_param_names(model)
     for name, param in model.named_parameters():
+        if efficient_finetune == 'bit_fit':
+            # For bit_fit, we disable tuning everything except the bias terms
+            if 'bias' not in name:
+                param.requires_grad = False
+        elif efficient_finetune == 'norm_fit':
+            # For norm-fit, we finetune all the normalization layers and bias layers
+            if name not in norm_param_names and 'bias' not in name:
+                param.requires_grad = False
+
```

Comment:
If the user provides a string not in [bit_fit, norm_fit], we need to raise errors, like how we did in the top-k average choices.

Reply:
Should be fine for now. We may still give user a fallback solution.